src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassUpdateServiceImpl.java

```diff
@@ -545,4 +551,15 @@ public void setMaxStatusCheckAttempts(int maxStatusCheckAttempts) {
         this.maxStatusCheckAttempts = maxStatusCheckAttempts;
     }
 
+    public void setDocumentProcessingQueue(DocumentProcessingQueue documentProcessingQueue) {
+        this.documentProcessingQueue = documentProcessingQueue;
+    }
+
+    public DocumentProcessingQueue getDocumentProcessingQueue() {
+        if (documentProcessingQueue == null) {
+            documentProcessingQueue = (DocumentProcessingQueue) SpringContext.getService(RassConstants.RICE_DOCUMENT_PROCESSING_QUEUE_SERVICE_NAME);
+        }
+        return documentProcessingQueue;
+    }
```

Comment:
Can RassUpdateServiceImpl be configured to have its service reference be set only by injection, and not by falling back to SpringContext calls?

If you're having trouble referencing the DocumentProcessingQueue bean by name in "cu-spring-rass.xml", you could try adding a bean-importing entry near the top of our "cu-spring-kfs.xml" file.

Reply:
I was having spring injection issues, that is why I switched to spring contect .... I will try doing the bein import in cu-spring-kfs